powershellgroup/powershellgroup.resource.ps1

```diff
@@ -152,6 +163,17 @@ elseif ($Operation -eq 'Get')
         {
             $inputht = @{}
             $ResourceTypeName = ($inputobj_pscustomobj.type -split "/")[1]
+
+            if ($r.ImplementationDetail -eq 'ScriptBased') {
```

Comment:
`$r` is not defined here.  
`$cachedResourceInfo` is probably what is meant here.